test/house_test.rb

```diff
@@ -0,0 +1,140 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+
+class HouseTest < Minitest::Test
+  def test_it_exists
+    house = House.new("$400000", "123 sugar lane")
+    assert_instance_of House, house
+  end
+
+  def test_it_has_a_price
+    house1 = House.new("$400000", "123 sugar lane")
+    assert_equal 400000, house1.price
+
+    house2 = House.new("250000", "124 sugar lane")
+    assert_equal 250000, house2.price
```

Comment:
This is fine that you tested two different houses, but unless there's something substantially different about the two of them, there's really no reason to do so.